crates/node/core/src/args/network.rs

```diff
@@ -148,9 +148,29 @@ pub struct NetworkArgs {
     /// Max capacity of cache of hashes for transactions pending fetch.
     #[arg(long = "max-tx-pending-fetch", value_name = "COUNT", default_value_t = DEFAULT_MAX_CAPACITY_CACHE_PENDING_FETCH, verbatim_doc_comment)]
     pub max_capacity_cache_txns_pending_fetch: u32,
+
+    /// Name of docker non-host interface used to communicate with peers.
+    ///
+    /// If flags is set, but no
+    /// value is passed, the default interface `eth0` is tried.
+    #[arg(long = "docker-if", conflicts_with = "addr")]
+    pub docker: Option<String>,
```

Comment:
this is not docker specific, is it? if not, we should not name it `--docker-if`. should also live under net, so its `--net.if`

Reply:
alright, in other words, peel off an abstraction layer. on it.